src/panels/lovelace/components/hui-image.ts

```diff
@@ -233,7 +311,7 @@ export class HuiImage extends LitElement {
     }
 
     // One the first render we will not know the width
-    const element_width = this._image.offsetWidth
+    const element_width = this._image?.offsetWidth
```

Comment:
`this._image?.offsetWidth ?? MAX_WIDTH`;

Reply:
done